scripts/update_configs.sh

```diff
@@ -17,9 +17,9 @@ if [ "$1" != "stable" ] && [ "$1" != "lts" ]; then
     echo $MSG && exit 1
 fi
 
-wget https://raw.githubusercontent.com/hyperledger/iroha/iroha2-stable/configs/peer/config.json -O "$(dirname "${BASH_SOURCE[0]}")/client_cli/$1/config.json"
-wget https://raw.githubusercontent.com/hyperledger/iroha/iroha2-lts/configs/peer/config.json -O "$(dirname "${BASH_SOURCE[0]}")/client_cli/$1/config.json"
+wget https://raw.githubusercontent.com/hyperledger/iroha/iroha2-stable/configs/peer/config.json -O "./configs/client/$1/config.json"
```

Comment:
Either one of them is brittle. We should probably agree on a directory structure. Namely we should probably put everything Iroha related into `.config` or some other standard XDG-compliant directory and atleast use `$HOME` or other env vars.

Reply:
or the script can take the path argument